js-jasmine/spec/gilded_rose_spec.js

```diff
@@ -1,74 +1,81 @@
-var {Shop, Item} = require('../src/gilded_rose.js');
+const BackstagePass = require('../src/backstage_passes')
+const AgedBrie = require('../src/aged_brie')
+const Sulfuras = require('../src/sulfuras')
+const NormalItem = require('../src/normal_item')
+const ConjuredItem = require('../src/conjured_item')
+const Item = require('../src/item')
+
 describe("Gilded Rose", function() {
 
-  // it("should foo", function() {
-  //   const gildedRose = new Shop([ new Item("foo", 0, 0) ]);
-  //   const items = gildedRose.updateQuality();
-  //   expect(items[0].name).toEqual("fixme");
-  // });
+  it('Normal items quality reduces in intervals of 1', function() {
+    let normalItem = new NormalItem([ new Item('test1', 5, 10) ])
```

Comment:
Looks like you have some good tests, one improvement could be to separate them out into their own spec files, for example a separate spec file for aged brie, sulfuras etc.